### PR TITLE
Documentation of three components

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Completed documentation for Grids , Input component and Navigation.

### CODE REVIEW

**Review by stutisk — APPROVED** [review]
Code looks clean ! Can go forward with merging it

**Review by vidhi-thakur — APPROVED** [review]
looks good to me

File: Docs/Components-pages/Grids-page.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                <div class="box-6 flex box"> Box-6 </div>
+            </div>
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522three-box-grid%2520grid%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-1%2520flex%2520box%2522%253E%2520Box-1%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-2%2520flex%2520box%2522%253E%2520Box-2%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-3%2520flex%2520box%2522%253E%2520Box-3%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-4%2520flex%2520box%2522%253E%2520Box-4%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-5%2520flex%2520box%2522%253E%2520Box-5%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-6%2520flex%2520box%2522%253E%2520Box-6%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 513px; height: 335px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (manojasarsa) [inline Docs/Components-pages/Grids-page.html:84]: You can avoid using inline-styling as it's not a good practice. Instead, you can make a class for iframe, and give the css for it.

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

File: Docs/Components-pages/Navigation-page.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <section class="content-subsection subsection flex" id="#installation">
+            <h2 class="component-demo-heading"> Simple Navigation </h2>
+
+            <nav class="nav-component">
+                <div>
+                    <a href=""> <img src="/components/images/Blossom.png" alt="Blossom store" class="logo-img"></a>
+        
+                </div>
+                <div class="nav-center">
+                    <input class="search-bar" type="text" placeholder="Search here..." />
```
Comment (MaruthiPothuganti) [inline Docs/Components-pages/Navigation-page.html:66]: This input can have a type="search"

File: components/Navigation/Navigation.css [hunk 3]
```diff
@@ -7,7 +7,6 @@
     justify-content: space-between;
     align-items: center;
     flex-wrap: nowrap;
```
Comment (MaruthiPothuganti) [inline components/Navigation/Navigation.css:9]: You can remove this. By default, **flex-wrap** has **nowrap** value

File: components/Navigation/Navigation.css [hunk 4]
```diff
@@ -7,7 +7,6 @@
...
-    
 }
 ion-icon{
```
Comment (MaruthiPothuganti) [inline components/Navigation/Navigation.css:11]: avoid styling with tag names

File: components/Navigation/Navigation.html [hunk 5]
```diff
@@ -42,8 +42,6 @@
         <div class="right-nav-menu">
             <ion-icon name="menu-outline"></ion-icon>
         </div>
-
-
     </nav>
    
     <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
```
Comment (MaruthiPothuganti) [inline components/Navigation/Navigation.html:47]: Place the icon script tags in the head section as icons will also parse with the HTML file.

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes. Everything else looks good.

**Pull request merged**